### DISCUSSION TITLE
Error when attempted to save page to Obsidian after successful setup

### DISCUSSION THREAD

**Discussion by tomstagg** [coddingtonbear/obsidian-web#171] [Bug Report]
### Describe the bug

After setup of the plugin and active local rest api on Obsidian, clicking the Take Note icon on a webpage produces the error: 
"Could not connect to Obsidian! Make sure Obsidian is running and that the Obsidian Local REST API plugin is enabled."

### To Reproduce

1. Setup Local Rest API and enabled non-encrypted server on port 27123
2. Configured chrome extension and validated setup with authorization bearer token, confirmed success
3. Validated APIs calls using `https://coddingtonbear.github.io/obsidian-local-rest-api/` setting up with non-secure mode, auth token and received valid 200 responses

Providing bug report here with details. 

UI Error: 

![image](https://github.com/coddingtonbear/obsidian-web/assets/7804088/18650dcf-87a7-4682-8ae7-8f34f3cea01e)

Bug Report: 
[obsidian-local-rest-api.bug_report.json](https://github.com/coddingtonbear/obsidian-web/files/14433691/obsidian-local-rest-api.bug_report.4.json)

Using version: 3.2.5 of the chrome extension

I wasn't sure if non-encrypted server on port 27123 is fully supported? 

### Expected Behavior

That the web URL is saved to my obsidian note.

### Operating System

sonoma 14.3.1

### Browser

Chrome 

### Browser Version

122.0.6261.69

**Comment by coddingtonbear** [comment]
HTTP (non-encrypted) connections should work just fine.  Looking at the connection settings in your bug report, it looks like it has selected the HTTPS (secure) mode of connecting to Obsidian, though.

When you're saying you're using the non-encrypted connection, are you saying that you have HTTPS disabled in Obsidian Local REST API, or are you saying you just haven't configured your operating system to trust the certificate?

**Reply by tomstagg** [reply]
I've gone down the non-encrypted mode as don't have full access to the operating system to add certificates. I'm not sure how to disable HTTPS in Obsidian Local REST API, though, can't see a setting for it. 

I've just enabled the non-encrypted (HTTP) Server:
![image](https://github.com/coddingtonbear/obsidian-web/assets/7804088/249b278f-559f-4a19-8371-7e69375c84df)

I get this in terminal:
``` bash
curl -X 'GET' \
  'http://127.0.0.1:27123/active/' \
  -H 'accept: text/markdown' \
  -H 'Authorization: Bearer <BEARER_TOKEN>' \
-o /dev/null -s -w "%{http_code}\n"
200
```

Looks like I'm close, but missing something!?

**Reply by tomstagg** [reply]
It's now working! I changed the encrypted port to 27125, so it doesn't hit the port for the server endpoint. Validated in the plugin options, that it was:

>  `connected insecurely to the API using HTTP` 


The red shield threw me as thought it was failing until I checked the hover-over statement. Then I restarted the Chrome extension and it works. 

The `README.md` states it supports port 27124 only, which could be changed to: 

```
  * Note: Supports use only with the encrypted (HTTPS) port 27124 or non-encrypted (HTTP) port 27123.
```
